### PR TITLE
Update delete publisher

### PR DESCRIPTION
**Pull request by hotuanhung**
## Related Tickets
[[Admin] Update nhà xuất bản](https://edu-redmine.sun-asterisk.vn/issues/67690)
[[Admin] Xóa nhà xuât bản](https://edu-redmine.sun-asterisk.vn/issues/67691)
## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
Screenshot from 2023-09-13 14-12-52
Screenshot from 2023-09-13 14-13-01
Screenshot from 2023-09-13 14-13-05
Screenshot from 2023-09-13 14-15-20
Screenshot from 2023-09-13 14-15-38
Screenshot from 2023-09-13 14-15-54
Screenshot from 2023-09-13 14-15-58

### CODE REVIEW

File: app/views/admin/publishers/_form.html.erb [hunk 1]
```diff
@@ -1,21 +1,45 @@
...
 
-  <div class="form-group">
-    <%= f.submit t("admin.book.button"), class: "btn btn-primary" %>
-  </div>
+    <div class="form-group">
+      <%= f.submit t("admin.book.button"), class: "btn btn-primary" %>
+    </div>
+  <% end %>
+<% else %>
+    <%= form_with(model: @publisher, url: admin_publisher_path(@publisher) , method: :patch) do |f| %>
```
Comment (ngocvt-0484) [inline app/views/admin/publishers/_form.html.erb:24]: không cần e. chỉ cần 1 form là đủ mà
e k cần điền method mà nó sẽ tự nhân

**Comment by hotuanhung** [comment]
ready

File: app/views/admin/publishers/_form.html.erb (hunk 1 above)
Comment (ngocvt-0484) [reply app/views/admin/publishers/_form.html.erb:24]: e đổi qua dùng form_for đi
form_for [:admin, @publisher] do |f|

**Comment by hotuanhung** [comment]
ready

**Pull request merged**